### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by krzyraj**

### CODE REVIEW

**Comment by krzyraj** [comment]
ping @DominikaEZ

**Comment by clash82** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -53,19 +53,19 @@ php ezpublish/console assets:install --symlink
 
 ## Usage
 
-Place following helper `{{ show_privacy_cookie_banner(%privacy_policy_url%) }}` somewhere in your footer template before body ending tag. Replace `%privacy_policy_url%` with your policy page address.
+Insert the following `{{ show_privacy_cookie_banner(%privacy_policy_url%) }}` helper somewhere in your footer template before body ending tag. Replace `%privacy_policy_url%` parameter with your policy page address.
```
Comment (DominikaK) [inline README.md]: +Insert the following `{{ show_privacy_cookie_banner(%privacy_policy_url%) }}` helper somewhere in your footer template before the body ending tag. Replace the `%privacy_policy_url%` parameter with your policy page address.

File: README.md [hunk 2]
```diff
@@ -53,19 +53,19 @@ php ezpublish/console assets:install --symlink
...
 
-Note that %privacy_policy_url% is not required, in this case no policy link will be shown.
+Note that `%privacy_policy_url%` parameter is not required, in this case no policy link will be shown.
```
Comment (DominikaK) [inline README.md]: +Note that the `%privacy_policy_url%` parameter is not required, in this case no policy link will be shown.

File: README.md [hunk 3]
```diff
@@ -82,4 +82,4 @@ or
 
 ##Abstraction
 
-Privacy Cookie Bundle v0.2 allows to increase abstraction level on the implementation side. Example of usage is shown on [EzDemoBundle](https://github.com/ezsystems/DemoBundle) (part of the eZ Publish / eZ Platform) in which banner options are retrieved from specified content type structure.
+Privacy Cookie Bundle v0.2 allows to increase abstraction level on the implementation side. Example of the usage can be see at [EzDemoBundle](https://github.com/ezsystems/DemoBundle). Banner options are retrieved from a specified content type structure.
```
Comment (DominikaK) [inline README.md]: +Privacy Cookie Bundle v0.2 allows increasing the abstraction level on the implementation side. Example of the usage can be see at [EzDemoBundle](https://github.com/ezsystems/DemoBundle). Banner options are retrieved from a specified Content Type structure.

**Comment by clash82** [comment]
Thanks @krzyraj and @DominikaEZ :)

**Pull request closed without merging**